config/BUILD.bazel

```diff
@@ -1,5 +1,23 @@
 package(default_visibility = ["//visibility:public"])
 
+sh_binary(
+    name = "mkpj_k8s",
```

Comment:
IMO this would be cleaner as //config:mkpj than mkpj_k8s because there's more than just k8s in this deployment (at least for now)